operators/src/engine/operator.rs

```diff
@@ -112,7 +112,7 @@ where
     }
 }
 
-impl<R, Q> InitializedOperator for Box<dyn InitializedOperatorB<R, Q>>
+impl<R, Q> InitializedOperatorBase for Box<dyn InitializedOperator<R, Q>>
 where
     R: Clone + 'static + ResultDescriptor,
     Q: Clone + 'static,
```

Comment:
Is this really a good bound for the `QueryProcessor`?

Reply:
51e1381cde7d2c2f6080c710fb06190362843ef2